web-app/js/ng-app/book/controllers/book.controller.js

```diff
@@ -1,15 +1,18 @@
 angular.module('ang').controller('BookController', [
-    '$route',
-    'bookService',
+    '$state',
+    'BookService',
     BookController
 ]);
 
-function BookController($route, bookService) {
+function BookController($state, BookService) {
     var vm = this;
+
     vm.reverse = false;
 
-    bookService.getAllBooks().then(function (response) {
+    BookService.getAllBooks().then(function (response) {
```

Comment:
Instances we use with small letter.

bookService.get...

You should to declare service like this:

.service('bookService', BookService);